apps/functions-subscription/src/adapters/azure/functions/codec.ts

```diff
@@ -56,3 +57,13 @@ export const toTrialAPI = (trial: Trial): TrialAPI => {
       }
     : trialAPI;
 };
+
+export const toTrialListAPI = (
+  trials: readonly Trial[],
+): TrialPaginatedCollection => {
+  return {
+    items: trials.map(toTrialAPI),
```

Comment:
**issue(blocking)**: I think this is not coherent with the OpenAPI

I suggest you to try writing an appropriate test first, to validate what I'm saying, but I think that the `toTrialListAPI` would return every properties of the `Trial` type, meanwhile the OpenAPI specification define that the `TrialSlim` object, which is the value of the elements within the `items` here, should contain only few of them